docs-dev/UPGRADING.md

```diff
@@ -0,0 +1,25 @@
+# Upgrade a Ceramic Node
+
+This document is a quick reference for minor version upgrades only, of a running ceramic node running in daemon mode with the package installed globally. 
+
+1) ssh into the server running the ceramic daemon
+
+2) upgrade the ceramic packages
+
+`npm install -g @ceramicnetwork/cli`
+
+If a specific version is desired, specify the normal way
+
+`npm install -g @ceramicnetwork/cli@2.43.0`
+
+Note that although this references the CLI package, the dependencies will cause the core and other server packages to update.
+
+3) restart the ceramic daemon
+
+Depending on how you have configured your server, this may involve a command such as
+
+`systemctl stop ceramic && systemctl start ceramic`
+
+Note this presumes that you have used systemctl startup scripts
+
```

Comment:
hey @kammerdiener can we get a copy of one of your systemctl startup scripts or where did those come from? 

 @3benbox what would be the one line command to upgrade the digitalocean kubernetes based install?  will it automatically upgrade like every time it restarts or?

Reply:
cool would it work with `npx @ceramicnetwork/cli daemon` without the version pin?  That way they could update without editing the script?

is npx installed with node or its a separate install?  ha sorry for lazy questions but just wanted to give docs for how to set up like you have it on hirenodes!

Maybe we should mention the ulimit too now that i think of it - that might be a separate doc for server-config-example ...

Reply:
When running with systemd, we should not use the root account. Any security issues with the ports we expose could escalate to a host compromise.

My ugly unit file is templated with Ansible in the simpledeploy repo
https://github.com/ceramicstudio/simpledeploy/blob/main/ansible/roles/ceramic/templates/ceramic.service.j2

Reply:
ok cool - linked them to SimpleDeploy and pointed out that our upgrade example is just a simplified example.